includes/Jobs/GenerateProductFeed.php

```diff
@@ -219,4 +269,43 @@ protected function get_batch_size(): int {
 		return apply_filters( 'facebook_for_woocommerce_feed_generation_num_products_per_batch', 15 );
 	}
 
+	/**
+	 * Track feed file generation time.
+	 *
+	 * @since x.x.x
+	 * @param Int $time Feed generation execution length.
+	 */
+	public function track_feed_file_generation_time( $time ) {
+		update_option( self::OPTION_FEED_GENERATION_TIME, $time );
+	}
+
+	/**
+	 * Get feed generation execution length.
+	 *
+	 * @since x.x.x
+	 */
+	public function get_feed_file_generation_time() {
```

Comment:
> These methods should all be private - they are internal to GenerateProductFeed, if other code used them it could impact the measurement.

I agree, at least the setters should be `private`